spec/requests/plate_picks_request_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe 'PlatePicks', type: :request do
```

Comment:
What is this test for?? Rubocop wingeing if class has no test?

Reply:
Scaffolded out, I should probably actually add the tests.